foxy/distribution.yaml

```diff
@@ -1012,6 +1012,21 @@ repositories:
       url: https://github.com/ros2/message_filters.git
       version: foxy
     status: maintained
+  mrpt2:
+    doc:
+      type: git
+      url: https://github.com/MRPT/mrpt.git
+      version: master
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/mrpt-ros2-pkg-release/mrpt2-release.git
+      version: 2.0.4-1
+    source:
+      type: git
+      url: https://github.com/MRPT/mrpt.git
+      version: develop
```

Comment:
Any reason that the source branch is different than the doc branch?  This is allowed, but is unusual.

Reply:
> Any reason...

=> A typo. Good eye!  :-) 

Fixed in a second commit.